python/SHIELD_DRT_ACCESS/SHIELD_DRT_ACCESS_test.py

```diff
@@ -0,0 +1,154 @@
+import sys
+import unittest
+try:
+    from unittest.mock import MagicMock
+except ImportError:
+    from mock import MagicMock
+import botocore
+
+##############
+# Parameters #
+##############
+
+# Define the default resource to report to Config Rules
+DEFAULT_RESOURCE_TYPE = 'AWS::::Account'
+
+#############
+# Main Code #
+#############
+
+CONFIG_CLIENT_MOCK = MagicMock()
+STS_CLIENT_MOCK = MagicMock()
+SHIELD_CLIENT_MOCK = MagicMock()
+
+class Boto3Mock():
+    @staticmethod
+    def client(client_name, *args, **kwargs):
+        if client_name == 'config':
+            return CONFIG_CLIENT_MOCK
+        if client_name == 'sts':
+            return STS_CLIENT_MOCK
+        if client_name == 'shield':
+            return SHIELD_CLIENT_MOCK
+        raise Exception("Attempting to create an unknown client")
+
+sys.modules['boto3'] = Boto3Mock()
+
+RULE = __import__('SHIELD_DRT_ACCESS')
+
+class SampleTest(unittest.TestCase):
+
+    event_drt_access = {'RoleArn':'arn:aws:iam::123456789012:role/test', 'LogBucketList':['some_string']}
+    event_drt_not_enabled = botocore.exceptions.ClientError({'Error':{'Code':'ResourceNotFoundException', 'Message': 'The subscription does not exist.'}}, 'operation')
```

Comment:
This variable can also be removed and its value can be directly passed as the side_effect parameter value.